dmd-frontend/src/components/Editor.tsx

```diff
@@ -13,51 +16,112 @@ const useStyles = makeStyles((theme: Theme) => ({
   textArea: {
     marginTop: '10px',
     marginBottom: '10px',
-    width: '100%',
+    minWidth: '100%',
+    maxwidth: '100%',
     fontSize: '14px',
     padding: '10px',
     fontFamily: 'Consolas,Courier,serif',
     lineHeight: '21px;',
     boxSizing: 'border-box',
   },
+  gridContainer: {
+    marginTop: '20px',
+  },
+  msgRequire:{
+    color: "red"
+  }
 }));
 
 interface comProps {
   className: string;
-  onSaveFunc: (term: string) => void;
-}
-
-interface comState {
-  text: string;
+  onSaveFunc: (title: string, content: string) => void;
 }
 
 const Editor = ({ className, onSaveFunc }: comProps) => {
   const classes = useStyles();
 
-  const [text, setText] = React.useState('');
+  const [titleValue, setTitleValue] = React.useState('');
+  const [contentValue, setContentValue] = React.useState('');
+  const { handleSubmit, register, errors} = useForm({
+    mode: "onChange"
+  });
+  const createDOMPurify = require('dompurify');
+  const purifyDOM = createDOMPurify(window);
 
-  const onFormSubmit = (e: React.ChangeEvent<HTMLFormElement>) => {
+  const onFormSubmit = React.useCallback((data, e) => {
     e.preventDefault();
-    onSaveFunc(text);
+    onSaveFunc(purifyDOM.sanitize(titleValue), purifyDOM.sanitize(contentValue));
```

Comment:
If a person want to add HTML tag in his markdown, does this remove these code?

E.g:
```
How to use <script> tag
```

Reply:
I'm sorry, I'm wrong
DOMpurify only applies to Title
I will fix it now

Reply:
What if the title is the same to example from above?

Reply:
Ok, I got it
I have not thought that Title needs this case
I will research how to encode